docs/USAGE.md

```diff
@@ -0,0 +1,80 @@
+# Usage
+
+- [How do I interact with the app?](#interacting-with-the-app)
+- [How do I stop and start the server?](#stopping-and-starting-the-server)
+- [How do I run tests?](#running-tests)
+- [How do I rebuild Ember?](#rebuilding-ember)
+- [Do I need special environment variables?](#environment)
+- [How do I generate and serve documentation?](#generating-documentation)
+- [How do I push changes to GitHub?](#pushing-changes)
```

Comment:
Any sections I've missed? Contribution types I didn't include? Other common tasks we want to talk about?